layer/index.js

```diff
@@ -89,7 +90,7 @@ router.route('/').post(auth0Middleware(), async (req, res) => {
 
         if (!project) return utils.respondWithError(res, 404, 'Project does not exist')
 
-        const newLayer = Layer.build(projectId, label, canvases)
+        const newLayer = Layer.build(projectId, label, canvases, req.user._id)
```

Comment:
creator isn't the req.user._id it is the agent from that token. It may be handy in the object, but can also be assembled without trouble as we worked hard to make the hexcodes match.